packages/create-mc-app/bin/create-mc-app.js

```diff
@@ -71,12 +71,18 @@ const packageManager = useYarn ? 'yarn' : 'npm';
 // TODO: we could check for min yarn/npm versions
 // See https://github.com/facebook/create-react-app/blob/0f4781e8507249ce29a9ac1409fece67c1a53c38/packages/create-react-app/createReactApp.js#L225-L254
 
-const templatePath = path.join(__dirname, '../templates', templateName);
+const templatePackageName = `@commercetools-frontend/application-template-${templateName}`;
+const modulePathForPackage = require.resolve
+  .paths(templatePackageName)
+  .find(modulesPath =>
+    fs.existsSync(path.join(modulesPath, templatePackageName))
```

Comment:
We only want to know in which `node_modules` folder the template package got installed.

If we were to use `require.resolve('@commercetools-frontend/application-template-starter')` it will fail because the function resolves the path to the module **entry point**. However, we are only interested in the **location** of the package.